test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetSourcesCommandTest.cs

```diff
@@ -156,6 +157,78 @@ public void SourcesCommandTest_EnableSource_WarnWhenUsingHttp()
             }
         }
 
+        [Theory]
+        [InlineData("http://source.test", "true", false)]
+        [InlineData("http://source.test", "false", true)]
```

Comment:
I think even this is redundant. I'm guessing there's already a test that asserts a warning is raised...when no disabled.

Reply:
Thanks! Yes, there are tests testing about enabling and disabling http source already.
I removed the two newly added tests entirely.